src/nr_openai_observability/bedrock.py

```diff
@@ -105,12 +107,12 @@ def patcher_bedrock_create_completion(original_fn, *args, **kwargs):
 
         with newrelic.agent.FunctionTrace(
             name=trace_name, group="", terminal=True
-        ):
+        ) as trace:
```

Comment:
did you mean to use `trace.add_custom_attribute(...)` here with completion_id? I see that you added it below for OpenAI.

Reply:
Womp womp. This is what I get for fixing merge conflicts in GitHub. Yep. Adding that back in